src/hb-ot-font.cc

```diff
@@ -301,6 +301,34 @@ struct hb_ot_face_cbdt_accelerator_t
   }
 };
 
+struct hb_ot_face_post_accelerator_t
+{
+  hb_blob_t *post_blob;
+  unsigned int post_len;
+  const OT::post *post;
+
+  inline void init (hb_face_t *face)
+  {
+    this->post_blob = OT::Sanitizer<OT::post>::sanitize (face->reference_table (HB_OT_TAG_post));
+    this->post = OT::Sanitizer<OT::post>::lock_instance (this->post_blob);
+    this->post_len = hb_blob_get_length (this->post_blob);
+  }
+
+  inline void fini (void)
+  {
+    hb_blob_destroy (this->post_blob);
+  }
+
+  inline bool get_glyph_name (hb_codepoint_t glyph,
+                              char *name, unsigned int size) const
+  {
+    if (unlikely (!name) || unlikely(!size))
```

Comment:
We just need to check for size.

Reply:
Humm.  Returning false doesn't sound quite right. Don't know.  Maybe doesn't matter.